cropper/src/main/java/com/canhub/cropper/CropImageContractOptions.kt

```diff
@@ -541,6 +542,20 @@ data class CropImageContractOptions @JvmOverloads constructor(
     fun setIntentChooserPriorityList(priorityAppPackages: List<String>) = cropImageOptions.apply {
         this.intentChooserPriorityList = priorityAppPackages
     }
+
+    /**
+     * Sets the background color of the Crop Image Activity screen.
+     */
+    fun setActivityBackgroundColor(@ColorInt color: Int) = cropImageOptions.apply {
+        this.activityBackgroundColor = color
+    }
+
+    /**
+     * Sets the toolbar color of the Crop Image Activity screen.
+     */
+    fun setToolbarColor(@ColorInt color: Int) = cropImageOptions.apply {
```

Comment:
We need to add this methods in our sample code

Reply:
Already done. I'll uncomment them as per your next comment. But let me know if we should keep that enabled for toolbar and let it ignore the primaryColor set in the default app styles.xml